src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md

```diff
@@ -0,0 +1,301 @@
+
+Data can be combined to identify an individual, but it can be difficult to work out whether new data adds to entropy. For example, knowing a person is from Australia doesn't reduce entropy if you already know the person is from Kangaroo Island.
+
+### Fingerprinting
+
+Techniques to identify and track the behaviour of individual users. Fingerprinting uses mechanisms that users aren't aware of and can't control. Sites such as [Panopticlick](https://panopticlick.eff.org) and [amiunique.org](https://amiunique.org/) show how fingerprint data can be combined to identify you as an individual.
+
+
+
+### Fingerprinting surface
+
+Something that can be used (probably in combination with other surfaces) to identify a particular user or device. For example, the `navigator.userAgent()` JavaScript method and the `User-Agent` HTTP request header provide access to a fingerprinting surface (the user agent string).
+
+### First-party
+
+Resources from the site you're visiting. For example, the page you're reading is on the site web.dev and includes resources from that site. See also [Third-party](#third-party).
+
+### Impression
+
+View of an ad. (See also [click-through rate](#click-through-rate-(ctr)).)
+
+### k-anonymity
+
+A measure of anonymity within a data set. If you have _k_ anonymity, you can't be distinguished from _k-1_ other individuals in the data set. In other words, _k_ individuals have the same information (including you).
+
+### Origin
+The origin of a request, including the server name but no path information. For example: `https://web.dev`.
+
+### Passive surfaces
+
+Some fingerprinting surfaces, such as user agent strings, IP addresses and accept-language headers, are available to every website whether the site asks for them or not. That means passive surfaces can easily consume a site's privacy budget.
+
+The Privacy Sandbox initiatives proposes replacing passive surfaces with active ways to get specific information, for example using Client Hints a single time to get the user's language rather than having an accept-language header for every response to every server.
+
+### Publisher
+
+The Privacy Sandbox explainers are mostly about ads, so the kinds of publishers referred to are ones that put ads on their web sites.
+
+### Reach
+
+The total number of people who see an ad.
+
+### Remarketing
+
+Advertising to people who've already visited your site. For example, an online store could show ads for a toy sale to people who previously viewed toys on their site.
+
+### Site
+
+(See above.)
```

Comment:
What is this supposed to be referring to?

Reply:
I'll do a commit to fix this. It should refer to the eTLD and Domain entries.